types/errors/abci_test.go

```diff
@@ -215,41 +238,15 @@ func TestABCIInfoSerializeErr(t *testing.T) {
 			debug: true,
 			exp:   fmt.Sprintf("%+v", myErrDecode),
 		},
-		// "multi error default encoder": {
```

Comment:
can we remove dead commented code?

Reply:
? I think I did just remove that as cleanup while working on this

Reply:
Sorry, long day...